packages/www/hooks/use-api.tsx

```diff
@@ -840,6 +853,83 @@ const makeContext = (state: ApiState, setState) => {
       return asset;
     },
 
+    async uploadAssets(files: File[]): Promise<void> {
+      const requestAssetUpload = async (
+        params
+      ): Promise<{ tusEndpoint: string }> => {
+        const [res, assetUpload] = await context.fetch(
+          `/asset/request-upload`,
+          {
+            method: "POST",
+            body: JSON.stringify(params),
+            headers: {
+              "content-type": "application/json",
+            },
+          }
+        );
+
+        if (!res.ok) {
+          throw new Error(assetUpload.errors.join(", "));
+        }
+        return assetUpload;
+      };
+
+      const updateStateWithProgressOrError = (
+        file: File,
+        progress?: number,
+        error?: Error
+      ) => {
+        setState((state) => ({
+          ...state,
+          currentFileUploads: {
+            ...state.currentFileUploads,
+            [file.name]: { file, progress, error },
+          },
+        }));
+      };
+
+      const getTusUpload = (file: File, tusEndpoint?: string) =>
+        new tus.Upload(file, {
```

Comment:
@victorges should I be using this endpoint? `tus` is mentioned in the docs, but might not be in production yet?

Reply:
I think we should! Worst thing that could happen would be some errors, but I think it's good dogfooding to the new resumable uploads API to use it from the dashboard.

Reply:
Sounds good!